azure/k8s/sealed-secrets/main-key-secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+data:
+  tls.crt: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUV6VENDQXJXZ0F3SUJBZ0lSQUkxMWZlZFlSa1NDbFBGSGRzSmdVcEl3RFFZSktvWklodmNOQVFFTEJRQXcKQURBZUZ3MHlNakEwTVRVd09UUXdNek5hRncwek1qQTBNVEl3T1RRd016TmFNQUF3Z2dJaU1BMEdDU3FHU0liMwpEUUVCQVFVQUE0SUNEd0F3Z2dJS0FvSUNBUURFUHladDJCZHlpU3lnSFRma21mVmw0cUlBU1ZsQ2F5T1dCZzM0CkRrSE5qTjdmc05GTTJWVXNaU3h1SURWUmFMOTZUWFVyRUE2TjQ5WXhXWU5tNEQrditaaDlyQzh1Vk01b0ExVWEKR3BnM29ZbW5ZcTdWVDRLMjFEV05sQjk5Z3B2UXpSSWFmSFYvS003bFYrc3dYQmVPMEF5TXBxY0FPYno3M1Iycwo4Y2xLdUpMdE1GSFZ6OW1EdjV4ZWtSNFNmTjg3WWwwaTNscjRXbnBMcW9GOUF1dmdRZE4xZ0FZTk9rWHVUOXpuCitZMTBRdldia3ZOZ2N3UHFNMi8yc1I4K295dFl2WGFDTWFVd1JZNFhyWlUwVGhvQXB5TityN0pBZERMQ3hnSVgKS2tOVHhPNVZJL3NqdERPc3hBWlJybnFnRVdDbkF0K0sxUXQvb1JLa0JyWjlSRVVzYjkrdHJKTFBSRDNGR2dUUAp6OUNieDVqSFpzMlVPWmRySXZaUlBtcFBFSjVhUk52MmpTNmFUYnQ2VHN0K1FJMytIems2SmptSWhWU0dHcnZzCnREbVl6RUlvWTRCWFBDWm83ZVhzYjRiTEg1M2RaQ2tGNElFOEgybGVNaEtCVk04Nnp4OUtDdGJRUTZyV01nU1oKSHNqR0xyMy9EcUpxYmtlaFdDMmt5aHphZ3Ewa3dGbGFVNlRZZ21YRkREU2UwNEFrT1JWVk1GUGxRdWVLSFRoMQpiZzJvVUNva1VrK1hnTFp2ckFKUklSeTZhMEFrWUNZS3hSV3NjQUJCbTFieUJPK00wUVYxZmxyOTgrdERsQ2FJClFIaHZFL0tEWWdNcEJQTE1TSmJ1cVVLK3NiRmN4dmlQa21qQkhqSVFwMXV1c1YySlQ5SXdqMmM5cDZVU1ZlNCsKY28rWXBRSURBUUFCbzBJd1FEQU9CZ05WSFE4QkFmOEVCQU1DQUFFd0R3WURWUjBUQVFIL0JBVXdBd0VCL3pBZApCZ05WSFE0RUZnUVVYVlhhZmJPK21xbm9qWndOYURpZHhWQjBIcmt3RFFZSktvWklodmNOQVFFTEJRQURnZ0lCCkFMSjZjSy9Uc3RtUVZSbkp2Y29HNUZQTGh4SXBqQkZheUFzc3c5MVdHdU9nTVhmVTJmQWNBcnNRY3Y3YzIvQUwKcVkwVFJlaklrdTFLQzlvcHRzdnV4QmFIMnEra3gydFFxSUltR0J0RlhZZmd0bVI3Mi8ybFNwZWUzTFZTODR2RQpOQmhhZHhvOEtRVmJiNkh0TE82Qml0ODFaZitYNnBxaWRNbklqeGFuUkJZZ2JGOVZ4Sm1EczlzenZtMDgwUjhyCkRtbWorU2NaalZ6cGtJaGpJeW5QNGU0UFA4YTd1dGR6bUNEcE5odWx3blNUN21SSk5mcitwV3FmRmRaaFBVSWwKbExlUWN3aEtmMllYbGpLeUtmUHNqWEhZblNKc1ZabVVwZ0ZUeWtzQVYzU0w5UzFIQWNZNUFWa0o1alc3MmU2QQprcVBUS1dESGZhU3BMOFNEQmh3UmhLNnhSN3l2ZHBmc0N1OFQzTmJaOG9lSWwzVk5HMitLTGNLQnlsRlZiSFFzCnhyWUFZSTRHa3Q1Mjk0cTdpTGo3OWNBN3RnL2N2NzJRY09TYlJGTTV5RFlweHNDOGVkdlZTRmFyd1hQNDNmSDMKTmNkaFpUK0FvQ0FQNHNJOHpJWks0SWpNaXRud2FremZsZHl6RU1OVm9sL3lwQlgrM01IRHcxOWdNSkdUNjNUTgp4S28zZW5FeWNoT25WU1c0OWwzaDlhQm9lMEhtRXQ3OVNZM0ZmYlVFM2hXVDBEZDQyb0xrTThuYTh6WU52R2JDClYxdXpZV2FGMk9qM1hmbmFqcTFZZytSdiswd1h0akZtZkxsUytQM09HYm50SERDdlpXYmtOYWVHUkFYcnlmL08KZUowQm1wMXBXMDgrQk56M21kZHZnS0dmMkdITjFZOThoNXcxTlNRbGw3azkKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
the same question here 🤔

Reply:
Same as above. I added a further change to this PR linked with a configuration problem that prevented the django app to connect to the database